sql/queries.sql

```diff
@@ -362,3 +362,236 @@ CREATE VIEW user_balance
+-- Search for a job by an employee.
+
+INSERT INTO search(search_id, user_id, input)
+	VALUES(7, 10, 'software developer');
+
+INSERT INTO job_seeker_search(job_seeker_id, search_id)
+	VALUES(10, 7);
+    
+-- -----------------------------------------------------------------------
+#ix
+-- Apply for a job by an employee
+
+INSERT INTO application(job_seeker_id, application_id, employer_id, posting_id)
+	VALUES(8, 5, 3, 2);
+     
+ -- -----------------------------------------------------------------------
+ #X
+ -- Accept/Deny a job offer by an employee
+ 
+ UPDATE application
+ SET status='accepted'
+ WHERE job_seeker_id=6 AND application.application_id=1;
+ 
+ -- -----------------------------------------------------------------------
+#xi
+-- Withdraw from an applied job by an employee
+
+UPDATE application
+ SET status='withdrawn'
+ WHERE job_seeker_id=6 AND application.application_id=1;
+ 
+-- -----------------------------------------------------------------------
+#xii
+-- Delete a profile by an employee
+
+DELETE FROM application
+WHERE job_seeker_id=6 AND application.application_id=1;
+
+-- -----------------------------------------------------------------------
+#xiii
+
+-- Report of applied jobs by an employee during a specific period of time
+-- (Job title, date applied, short description of the job up to 50 characters,
+-- status of the application).
+
+
+SELECT posting.title, posting.date_posted, posting.description, application.status
+FROM posting p, application app
+WHERE p.posting_id=app.posting_id AND app.job_seeker_id=6 AND
+ date_posted BETWEEN '2019-01-01' AND CURDATE();
+
+-- -----------------------------------------------------------------------
+#xiv
+-- Add/Delete/Edit a method of payment by a user
+
+-- Adding another method of payment
+INSERT INTO payment_method (user_id, method_id, account_number)
+    VALUES (1, 2, '34-175-9681');
+
+-- EDITING A METHOD OF PAYMENT
+UPDATE payment_method
+SET account_number='24-175-9681'
+WHERE user_id=1 AND method_id=2;
+
+-- DELETING A METHOD OF PAYMENT
+DELETE FROM payment_method
+WHERE user_id=1 AND method_id=2;
+
+-- -----------------------------------------------------------------------
+#xv
+-- Add/Delete/Edit an automatic payment by a user.
+
+-- ADDING AND EDITING AN AUTOMATIC PAYMENT
+UPDATE payment_method
+SET is_automatic=TRUE 
+WHERE user_id=1 AND method_id=1;
+
+-- DELETING AN AUTOMATIC PAYMENT
+UPDATE payment_method
+SET is_automatic=FALSE 
+WHERE user_id=1 AND method_id=1;
+
+-- -----------------------------------------------------------------------
+#xvi
+-- Make a manual payment by a user
+
+INSERT INTO payment (payment_id, user_id, payment_amount)
+    VALUES (9, 4, 1, 20);
+
+-- -----------------------------------------------------------------------
+#xvii
+-- Report of all users by the administrator for employers or employees
+-- (Name, email, category, status, balance)
+
+SELECT 'employer' AS employer, user_id, employer_name, email, employer_tier, SUM(bill_amount)
```

Comment:
user_id needs to be users.user_id; Balance is a function of bill_amount and payment_amount, not bill_amount alone.